src/pkg/store/backup.go

```diff
@@ -81,44 +79,18 @@ func (w Wrapper) GetBackups(
 
 // DeleteBackup deletes the backup and its details entry from the model store.
 func (w Wrapper) DeleteBackup(ctx context.Context, backupID model.StableID) error {
-	deets, _, err := w.GetDetailsFromBackupID(ctx, backupID)
```

Comment:
Are we okay with orphaning details snapshots when the associated backup gets deleted?

Reply:
Ah no - I need to wire up the call to `DeleteDetails` in the backup delete calls (in CLI).